docs/src/pages/configurations/custom-webpack-config/index.md

```diff
@@ -207,61 +205,36 @@ Nevertheless, edit `config` with care. Make sure to preserve the following confi
 Furthermore, `config` requires the `HtmlWebpackplugin` to generate the preview page, so rather than overwriting `config.plugins` you should probably append to it (or overwrite it with care), see [Issue #6020](https://github.com/storybookjs/storybook/issues/6020) for examples:
 
 ```js
-module.exports = async ({ config, mode }) => {
-  config.plugins.push(...)
-  return config;
-}
-```
-
-Finally, if your custom webpack config uses a loader that does not explicitly include specific file extensions via the `test` property, it is necessary to `exclude` the `.ejs` file extension from that loader.
-
-### Extend Mode (**Deprecated**)
```

Comment:
This is still supported in `webpack.config.js` though right?

Reply:
yes, we support it.. but I think removing it from the docs, would be best.

Reply:
Yeah, probably, just wanted to check